buildSrc/src/main/kotlin/kotlin-jvm-conventions.gradle.kts

```diff
@@ -4,37 +4,43 @@
 
 // Platform-specific configuration to compile JVM modules
 
-apply plugin: 'org.jetbrains.kotlin.jvm'
+import org.gradle.api.*
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+plugins {
+    kotlin("jvm")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_6
+    targetCompatibility = JavaVersion.VERSION_1_6
+}
 
-if (rootProject.ext.jvm_ir_enabled) {
-    kotlin.target.compilations.all {
+if (rootProject.extra.get("jvm_ir_enabled") as Boolean) {
+    kotlin.target.compilations.configureEach {
```

Comment:
@turansky please elaborate on using `configureEach` instead of `all`

Reply:
This change created according [Gradle migration guide](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:old_vs_new_configuration_api_overview)

Reply:
<img width="512" alt="Screen Shot 2021-01-18 at 7 06 19 PM" src="https://user-images.githubusercontent.com/9942723/104938517-428c0e80-59c0-11eb-98f3-6eac84c8da94.png">

Reply:
Okay, thanks!